irma/cmd/config.go

```diff
@@ -96,6 +96,9 @@ func configureIRMAServer() (*server.Configuration, error) {
 
 		conf.RedisSettings.TLSCertificate = viper.GetString("redis_tls_cert")
 		conf.RedisSettings.TLSCertificateFile = viper.GetString("redis_tls_cert_file")
+		conf.RedisSettings.TLSClientKeyFile = viper.GetString("redis_tls_client_key_file")
+		conf.RedisSettings.TLSClientCertificateFile = viper.GetString("redis_tls_client_cert_file")
+
```

Comment:
Can you also add the new option to the CLI help command? Then it's clear for everyone that these options exist.

https://github.com/privacybydesign/irmago/blob/0b3390be045b38b904358bceace229a413824b0b/irma/cmd/server.go#L124
https://github.com/privacybydesign/irmago/blob/0b3390be045b38b904358bceace229a413824b0b/irma/cmd/keyshare-server.go#L68
https://github.com/privacybydesign/irmago/blob/0b3390be045b38b904358bceace229a413824b0b/irma/cmd/keyshare-myirma.go#L71